### ISSUE TITLE
Joining the same conference when using different TS3-Server IPs.

### ISSUE THREAD

**Issue by mfreiholz** [mfreiholz/ocs#21]
Regarding to hyprdmg's problem, a solution is required.
(http://forum.teamspeak.com/showthread.php/111028-release-ts3video/page4)

## Problem
Since the TeamSpeak's IP address is used to generate a unique conference-name, it is a problem, if the same server can be reached with different IP addresses.

It's probably often the case that one user hosts everything local (127.0.0.1) and the other participants join with an external IP address. It's not always possible that the host uses the external IP address, too. Some routers block outgoing traffic to it's own external IP address.

## Solution
No idea yet...

### References
Router info by Bluescream: http://forum.insanefactory.com/discussion/5/connecting-from-your-pc-to-your-wan-ip#latest

**Comment by jacobq11** [comment]
Has there been a fix developed yet to the empty room situation from different ip's?

**Comment by kakomanda** [comment]
Yeah. I'm still waiting too. Isn't there a way to force everybody on the server to the same room?

**Comment by mfreiholz** [comment]
push... for myself...
I'm going to work on this soon.
